frontend/src/scenes/pipeline/hogfunctions/HogFunctionConfiguration.tsx

```diff
@@ -294,6 +300,44 @@ export function HogFunctionConfiguration({ templateId, id }: { templateId?: stri
                                     This destination will be triggered if <b>any of</b> the above filters match.
                                 </p>
                             </div>
+                            <div className="border bg-bg-light rounded p-3 space-y-2">
+                                <LemonLabel>Expected volume</LemonLabel>
+                                {sparkline ? (
+                                    <>
+                                        {sparkline.count > EVENT_THRESHOLD_ALERT_LEVEL ? (
+                                            <LemonBanner type="warning">
+                                                <b>Warning:</b> This destionation would have triggered{' '}
+                                                <strong>
+                                                    {sparkline.count ?? 0} time{sparkline.count !== 1 ? 's' : ''}
+                                                </strong>{' '}
+                                                in the last 7 days. Consider the impact of this function on your
+                                                infrastructure.
+                                            </LemonBanner>
+                                        ) : (
+                                            <p>
+                                                This destination would have triggered{' '}
+                                                <strong>
+                                                    {sparkline.count ?? 0} time{sparkline.count !== 1 ? 's' : ''}
+                                                </strong>{' '}
+                                                in the last 7 days.
+                                            </p>
+                                        )}
+                                        <div className="relative">
+                                            {sparklineLoading ? <Spinner className="absolute bottom-0 left-0" /> : null}
+                                            <Sparkline
+                                                type="bar"
+                                                className="w-full"
+                                                data={[{ name: 'Matching events', values: sparkline.data }]}
```

Comment:
Not sure how we would do this but it would be cool to color code - green if under threshold, red if over or something

Reply:
Hmm. Yeah, I'm not sure what's a good thing here. I could make it uniformly red or orange if over a threshold:
 
<img width="545" alt="image" src="https://github.com/user-attachments/assets/e2762e10-0a83-4e27-a8d3-ef4514864887">

Currently the threshold is calculated at 8000 events per week, and I don't want a red/green flip at 1142 events per day. I could change the threshold to trigger if any day is over 1000, but then what if all other days are very low? I'm getting a slight paralysis by overanalysis, so let's merge as is and adjust when we know more about our actual limits 🤷